iam-login-service/src/main/java/it/infn/mw/iam/config/SecurityConfig.java

```diff
@@ -922,6 +922,76 @@ protected void configure(HttpSecurity http) throws Exception {
     }
   }
 
+  @Configuration
+  @Order(30)
+  public static class AccountSearchApiEndpointConfig extends WebSecurityConfigurerAdapter {
```

Comment:
Unify the security configuration for account & group search and add tests that check that the configuration is correctly enforced (there are no such tests AFAICS)